usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -127,6 +127,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/recipient/count/](/api/v2/recipient/count/)|POST| Returns the count of recipents for the given filters |
 |[/api/v2/recipient/duns/<HASH_VALUE\>/](/api/v2/recipient/duns/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/)|GET| Returns a high-level overview of a specific recipient, given its id |
 |[/api/v2/recipient/duns/](/api/v2/recipient/duns/)|POST| Returns a list of recipients in USAspending DB |
+|[/api/v2/recipient/<RECIPIENT_ID\>/](/api/v2/recipient/recipient_id)|GET| Returns an individual recipient in USAspending DB |
```

Comment:
Similar to the original endpoint will need to update the example endpoint to include a hash.

Row I am referencing ```|[/api/v2/recipient/duns/<HASH_VALUE\>/](/api/v2/recipient/duns/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/)|GET| Returns a high-level overview of a specific recipient, given its id |```

The difference is in the endpoint linked in the parenthesis.

On the API UI the difference is 
> "detail": "ID ('<built-in function hash>') doesn't include Recipient-Level"

vs

> "detail": "Recipient ID not found: '99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R'."

On your local environment this is the URL you can use to see the deprecated endpoint message with that hash: http://localhost:8000/api/v2/recipient/duns/99a44eeb-23ef-e7c4-1f84-9a695b6f5d2e-R/